@here/harp-mapview/test/PoiRendererTest.ts

```diff
@@ -97,4 +103,88 @@ describe("PoiRenderer", function () {
             expect(screenBox.h).to.equal(16);
         });
     });
+
+    describe("search for cached images", function () {
+        const webGLRenderer = {
+            capabilities: {
+                isWebGL2: false
+            }
+        } as THREE.WebGLRenderer;
+        const mapViewStub = sinon.createStubInstance(MapView);
+        const poiManager = new PoiManager((mapViewStub as any) as MapView);
+        const testImageName = "testImage";
+        const mapEnv = new Env();
```

Comment:
move this to a beforeEach() function. Each test should start with a clean slate.